cxplugin-server/src/main/resources/buildServerResources/editRunParams.jsp

```diff
@@ -369,7 +369,7 @@ optionsBean.testConnection(cxServerUrl, cxUsername, cxPassword)}
         <td>
             <props:selectProperty name="${optionsBean.teamId}" className="longField">
```

Comment:
"longField" is that for number type values? Now we changing it to string type. check if that matters.